packages/ibm-products-styles/src/components/CreateFullPage/_create-full-page.scss

```diff
@@ -21,8 +21,15 @@
 $block-class: #{c4p-settings.$pkg-prefix}--create-full-page;
 $step-block-class: #{c4p-settings.$pkg-prefix}--create-full-page__step;
 
-.#{$block-class} .#{$block-class} {
+.#{$block-class}.#{$block-class} {
```

Comment:
Duplicate class names?

Reply:
@paul-balchin-ibm I didn't add the duplicate names, just made them functional assuming they were supposed to.

Removed the duplicate.